portfolio/src/main/webapp/style.css

```diff
@@ -280,3 +280,53 @@ section {
   font-family: "Fira Sans", serif;
   font-style: italic;
 }
+/* ------Contact page------ */
+#contact {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+#contact h1 {
+  position: absolute;
+  transform: translate(90px, -300px);
+  font-size: 50px;
+}
+.container {
+  transform: translate(90px);
+  width: 500px;
+  border-radius: 20px;
+  background-color: #f2f2f25a;
+  padding: 20px;
+  box-shadow: 10px 10px #0000005b;
+}
```

Comment:
Something I didn't mention before: ordering the css styling alphabetically can help readers search for styling a bit easier. Definitely a personal preference, and not required for SPS, but just wanted to share.